src/Action/Iterator.php

```diff
@@ -146,11 +146,11 @@ protected function match(array $row, AbstractScope $scope)
         return $match;
     }
 
-    protected function where($v1, $operator, $v2)
+    protected function evaluate($v1, $operator, $v2): bool
```

Comment:
what evaluate? `evaluateWhere` is fine...

Reply:
evaluateIf ?

Reply:
we do not evaluate Iterator/self, nothing more...